helpers/comment.ts

```diff
@@ -0,0 +1,26 @@
+import * as Octokit from '@octokit/rest';
+import { exit } from './utils';
+
+const createComment = async ({
+  octokit,
+  owner,
+  repo,
+  message,
+  issue_number: issueNumber,
+}: {
+  octokit: Octokit;
+  owner: string;
+  repo: string;
+  message: string;
+  issue_number: string;
+// eslint-disable-next-line max-len
+}): Promise<void | Octokit.Response<Octokit.IssuesCreateCommentResponse>> => octokit.issues.createComment({
```

Comment:
By using the _right_ type here, it means I don't have to have it locally - which is nice